js/os.ts

```diff
@@ -33,14 +33,22 @@ function setEnv(key: string, value: string): void {
  * the process. The environment object will only accept `string`s
  * as values.
  *
+ *       console.log(Deno.env("SHELL"));
  *       const myEnv = Deno.env();
  *       console.log(myEnv.SHELL);
  *       myEnv.TEST_VAR = "HELLO";
  *       const newEnv = Deno.env();
  *       console.log(myEnv.TEST_VAR == newEnv.TEST_VAR);
  */
-export function env(): { [index: string]: string } {
+export function env(): { [index: string]: string };
+export function env(key: string): string | undefined;
+export function env(
```

Comment:
Is third overloaded needed? Seems you're exhausting all options with first and second

Reply:
The last overload is special because it isn’t part of the function’s type signature, and its type must match all of the overloads.

Reply:
Oh I didn't know that. TIL, thanks!